compiler/src/dotty/tools/repl/ReplDriver.scala

```diff
@@ -264,7 +280,7 @@ class ReplDriver(settings: Array[String],
       name.startsWith(str.REPL_RES_PREFIX) && hasValidNumber && sym.info == defn.UnitType
     }
 
-    def displayMembers(symbol: Symbol) = if (tree.symbol.info.exists) {
+    def displayMembers(symbol: Symbol): (State, Seq[Diagnostic]) = if (tree.symbol.info.exists) {
```

Comment:
Maybe rename this to something else like `formatMembers` since it doesn't directly display anything anymore?